pandas-stubs/core/frame.pyi

```diff
@@ -1955,7 +1955,7 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def rolling(
         self,
-        window: int | str | BaseOffset | BaseIndexer,
+        window: int | str | Timedelta | BaseOffset | BaseIndexer,
```

Comment:
If `datetime.timdelta` is also allowed (I'm not sure), you could replace `pd.Timedelta` with `datetime.timdelta`. I think `pd.Timedelta` inherits from `datetime.timdelta`

Reply:
Yes, this is correct, so can you change all the`Timedelta` declarations you added to `rolling` to `timedelta` and change test as indicated below?